src/olympia/editors/tests/test_helpers.py

```diff
@@ -501,6 +509,49 @@ def test_nomination_to_public(self, sign_mock):
         self._check_score(amo.REVIEWED_ADDON_FULL)
 
     @patch('olympia.editors.helpers.sign_file')
+    def test_public_addon_with_version_awaiting_review_to_public(
```

Comment:
😿 I think `test_pending_to_sandbox` and `test_pending_to_public` are supposed to cover this but both of them are incorrectly setting the addon status to `amo.STATUS_NOMINATED` instead of `amo.STATUS_PUBLIC` - and not making sure there is an existing approved version to make it a valid addon.